packages/cli/src/TestWebhooks.ts

```diff
@@ -58,69 +67,54 @@ export class TestWebhooks implements IWebhookManager {
 		response: express.Response,
 	): Promise<IResponseCallbackData> {
 		const httpMethod = request.method;
-		let path = request.params.path;
 
-		// Reset request parameters
+		let path = request.params.path.endsWith('/')
+			? request.params.path.slice(0, -1)
+			: request.params.path;
+
```

Comment:
considering that we do this for test and prod webhooks, maybe we should move this bit to `WebhookHelpers.webhookRequestHandler`, and start passing a context object into `executeWebhook` calls.

Reply:
Not quite familiar yet with `WebhookHelpers` to address quickly, I'll come back to this later if needed.